internal/pkg/values/timespan.go

```diff
@@ -75,7 +75,38 @@ func (t *timeSpans) Set(value string) error {
 }
 
 func (t *timeSpans) String() string {
-	return fmt.Sprint(*t)
+	var stringBuilder strings.Builder
+
+	for i, timespan := range *t {
+		if i > 0 {
+			stringBuilder.WriteString(", ")
+		}
+
+		switch span := timespan.(type) {
+		case absoluteTimeSpan:
+			stringBuilder.WriteString(fmt.Sprintf("AbsoluteTimeSpan(%s - %s)", span.from.Format(time.RFC3339), span.to.Format(time.RFC3339)))
+		case *relativeTimeSpan:
+			weekdayFrom := span.weekdayFrom.String()[:3] // Get first 3 letters of the weekday
+			weekdayTo := span.weekdayTo.String()[:3]     // Get first 3 letters of the weekday
+			timeFrom := span.timeFrom.Format("15:04")    // Format time to "HH:MM"
+			timeTo := span.timeTo.Format("15:04")        // Format time to "HH:MM"
+			stringBuilder.WriteString(fmt.Sprintf(
+				"RelativeTimeSpan(%s-%s %s-%s %s)",
+				weekdayFrom,
+				weekdayTo,
+				timeFrom,
+				timeTo,
+				span.timezone.String()))
```

Comment:
we shouldnt allocate all of these values. just use them directly.
```suggestion
			weekdayFrom := span.weekdayFrom.String()[:3] // Get first 3 letters of the weekday
			weekdayTo := span.weekdayTo.String()[:3]     // Get first 3 letters of the weekday
			timeFrom := span.timeFrom.Format("15:04")    // Format time to "HH:MM"
			timeTo := span.timeTo.Format("15:04")        // Format time to "HH:MM"
			stringBuilder.WriteString(fmt.Sprintf(
				"relativeTimeSpan(%.3s-%.3s %s-%s %s)",
				span.weekdayFrom,
				span.weekdayTo,
				span.timeFrom.Format("15:04"),
				span.timeTo.Format("15:04"),
				span.timezone))
```